components/mosq/examples/broker/README.md

```diff
@@ -0,0 +1,48 @@
+# MQTT broker example
+
+## Overview
+
+This example runs a TCP broker on a specified host and port.
+
+### How to use this example
+
+Choose the preferred connection in the project configuration menu, build and run the example.
+After it connects vias the specified network interface, it start the mosquitto broker, which by default listens on the connected interface.
```

Comment:
A small typo "vias"
"After it connects via the specified network interface, it starts the mosquitto broker, which by default listens on the connected interface." ?

Reply:
will fix

Reply:
fixed and updated the README. Thanks!